yt/utilities/lib/pixelization_routines.pyx

```diff
@@ -1023,11 +1023,11 @@ def pixelize_sph_kernel_projection(
             h_j2 = fmax(hsml[j]*hsml[j], dx*dy)
             ih_j2 = 1.0/h_j2
 
-            w_j = pmass[j] / pdens[j] / hsml[j]**3
+            prefactor_j = pmass[j] / pdens[j] / hsml[j]**2
             if weight_field is None:
-                coeff = w_j * hsml[j] * quantity_to_smooth[j]
+                prefactor_j *= quantity_to_smooth[j]
             else:
-                coeff = w_j * hsml[j] * quantity_to_smooth[j] * _weight_field[j]
+                prefactor_j *= quantity_to_smooth[j] * _weight_field[j]
 
```

Comment:
I removed the misleading use of `w` here, as this is a scatter smoothing method, the prefactor only depends on j, hence the name `prefactor_j`

Reply:
I'm now much happier with this notation now, good work @AshKelly!

Reply:
Great, thank you very much!